resources/views/pages/ticket/[ticket].blade.php

```diff
@@ -0,0 +1,423 @@
+
+                <div class="flex mt-2 w-full justify-between border-embed-highlight items-center">
+                    @if ($existingUnlock)
+                        @php $unlockedAt = $existingUnlock->unlocked_hardcore_at ?? $existingUnlock->unlocked_at; @endphp
+                        @if ($existingUnlock->unlocker_id)
+                            <span>Manually unlocked by {!! userAvatar(User::firstWhere('id', $existingUnlock->unlocker_id), icon:false) !!} at {{ getNiceDate($unlockedAt->unix()) }}</span>
+                        @else
+                            {{ $ticket->reporter->User }} earned this achievement at 
+                            {{ getNiceDate($unlockedAt->unix()) }}
+                            @if ($unlockedAt > $ticket->ReportedAt)
+                                (after the report)
+                            @else
+                                (before the report)
+                            @endif
+                        @endif
+                    @else
+                        <div class="flex w-full justify-between border-embed-highlight items-center">
+                            {{ $ticket->reporter->User }} did not earn this achievement
+                            @if ($permissions >= Permissions::Moderator)
+                                <script>
+                                    function AwardManually(hardcore) {
+                                        showStatusMessage('Awarding...');
+
+                                        $.post('/request/user/award-achievement.php', {
+                                            user: '{{ $ticket->reporter->User }}',
+                                            achievement: {{ $ticket->achievement->id }},
+                                            hardcore: hardcore
+                                        })
+                                        .done(function () {
+                                            location.reload();
+                                        });
+                                    }
+                                </script>
+                                <div class="flex items-center gap-x-3">
+                                    <button class="btn" onclick="AwardManually(0)">Award Softcore</button>
+                                    @if ($ticket->Hardcore)
+                                    <button class="btn" onclick="AwardManually(1)">Award Hardcore</button>
+                                    @endif
+                                </div>
+                            @endif
+                        </div>
+                    @endif
+                </div>
+            </div>
+        </div>
+    @endif
+
+    <div class="mt-2">
+        <div class="flex w-full justify-between border-embed-highlight items-center">
+            <p role="heading" aria-level="2" class="mb-0.5 text-2xs font-bold">
+                Comments
+            </p>
+            @if ($permissions >= Permissions::Developer)
```

Comment:
We should definitely use a policy here with `@can()`. Maybe:
```php
@can('manage', App\Models\Ticket::model)
```

The only thing that gives me pause is in `TicketPolicy::manage()`, I see:

```php
public function manage(User $user): bool
{
    return $user->hasAnyRole([
        Role::HUB_MANAGER,
        Role::DEVELOPER_STAFF,
        Role::DEVELOPER,
        Role::DEVELOPER_JUNIOR,
    ])
        || $user->getAttribute('Permissions') >= Permissions::JuniorDeveloper;
}
```

The `@if` currently restricts against Jr Devs. Maybe it shouldn't? Or maybe we should have a new function in our ticket policy that excludes Jr Devs from the ability to view this data?

Reply:
This just hides the "Contact the Reporter" button from non-developers. I don't see any reason why it shouldn't be available for Junior Developers.

Reply:
Yes, my focus was more focused on the `manage()` policy itself, which implies broader reach than what I think we might want to allow for Junior Developers.

Reply:
After rethinking this, it's probably not an issue.